_includes/releases/v22.1/v22.1.16.md

```diff
@@ -0,0 +1,42 @@
+## v22.1.16
+
+Release Date: March 3, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-1-16-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+
+<h3 id="v22-1-16-sql-language-changes">SQL language changes</h3>
+
+- Added a hard limit to the amount of data that can be flushed to system tables for SQL stats. [#97401][#97401]
+
+<h3 id="v22-1-16-operational-changes">Operational changes</h3>
+
+- A [`BACKUP`](../v22.1/backup.html) which encounters too many retryable errors will now fail instead of pausing to allow subsequent backups the chance to succeed. [#96715][#96715]
+
+<h3 id="v22-1-16-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug in [Enterprise changefeeds](../v22.1/change-data-capture-overview.html) where long-running initial scans will fail to generate a checkpoint. Failure to generate a checkpoint is particularly bad if the changefeed restarts for any reason. Without checkpoints, the changefeed will restart from the beginning, and in the worst case, when exporting substantially sized tables, changefeed initial scans may have hard time completing. [#97052][#97052]
+- Fixed a bug where the [`SHOW GRANTS FOR public`](../v22.1/show-grants.html) command would return an error saying that the `public` role does not exist. [#96999][#96999]
+- The following spammy log message was removed: `> lease [...] expired before being followed by lease [...]; foreground traffic may have been impacted`. [#97378][#97378]
+- Fixed a bug in the query engine that could cause incorrect results in some cases when a [zigzag join](../v22.1/cost-based-optimizer.html#zigzag-joins) was planned. The bug could occur when the two indexes used for the zigzag join had a suffix of matching columns but with different directions. For example, planning a zigzag join with `INDEX(a ASC, b ASC)` and `INDEX(c ASC, b DESC)` could cause incorrect results. This bug has existed since at least [v19.1](../releases/index.html#v19-1). The optimizer will no longer plan a zigzag join in such cases. [#97440][#97440]
+- Added support for disabling cross-descriptor validation on lease renewal, which can be problematic when there are lots of descriptors with lots of foreign key references, in which cases, the cross-reference validation could starve schema changes. This can be enabled with `sql.catalog.descriptor_lease_renewal_cross_validation`. [#97644][#97644]
+- Columns referenced in partial index predicates and partial unique constraint predicates can no longer be dropped. The [`ALTER TABLE .. DROP COLUMN`](../v22.1/drop-column.html) statement now returns an error with a hint suggesting to drop the indexes and constraints first. This is a temporary safe-guard to prevent users from hitting #96924. This restriction will be lifted when that bug is fixed. [#97663][#97663]
```

Comment:
Suggest making '#96924 a link to https://github.com/cockroachdb/cockroach/issues/96924 so users can follow up more easily